core/src/main/java/org/bitcoinj/crypto/ECKey.java

```diff
@@ -152,9 +155,9 @@ public class ECKey implements EncryptableItem {
     @Nullable protected final BigInteger priv;  // A field element.
     protected final LazyECPoint pub;
 
-    // Creation time of the key in seconds since the epoch, or zero if the key was deserialized from a version that did
+    // Creation time of the key, or null if the key was deserialized from a version that did
     // not have this field.
-    protected long creationTimeSeconds;
+    @Nullable protected Instant creationTime = null;
```

Comment:
Can this be `final` and constructors with a creation instant parameter be added. Since we're refactoring here can we make things less mutable?  (Perhaps in a separate PR)

Reply:
I'm open for trying this out, but in a different PR.